src/main/java/com/google/firebase/auth/multitenancy/FirebaseTenantClient.java

```diff
@@ -58,10 +61,22 @@
         "Project ID is required to access the auth service. Use a service account credential or "
             + "set the project ID explicitly via FirebaseOptions. Alternatively you can also "
             + "set the project ID via the GOOGLE_CLOUD_PROJECT environment variable.");
-    this.tenantMgtBaseUrl = String.format(ID_TOOLKIT_URL, "v2", projectId);
+    this.tenantMgtBaseUrl = buildTenantMgtBaseUrl(projectId);
     this.httpClient = new AuthHttpClient(jsonFactory, requestFactory);
   }
 
+  private String buildTenantMgtBaseUrl(String projectId) {
```

Comment:
Nit: Make this name consistent with the method in `FirebaseUserManager.java`, ie. `getTenantMgtBaseUrl` (or vice versa)